docs/source/ja/model_doc/byt5.md

```diff
@@ -0,0 +1,154 @@
+<!--Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# ByT5
+
+## Overview
+
+ByT5 モデルは、[ByT5: 事前トレーニングされたバイト間モデルによるトークンフリーの未来へ](https://arxiv.org/abs/2105.13626) で、Linting Xue、Aditya Barua、Noah Constant、Rami によって提示されました。アル・ルフォウ、シャラン・ナラン、ミヒル
```

Comment:
I noticed in some of the translations, you don't translate the paper title and in others it is translated. I think for consistency, let's keep the paper titles untranslated.

Reply:
got it